guides/advanced_guides/filtering.md

```diff
@@ -110,6 +114,15 @@ Say you want to only show to your user movies that were released after a certain
 release_date > 795484800 // march 18, 1995
 ```
 
+Querying on `Avengers`, the above example results in the following CURL command:
+
+```bash
+$ curl \
+  -X GET 'http://localhost:7700/indexes/movies/search?' \
+    -d 'q=Avengers' \
+    -d 'filters=release_date > 795484800'
```

Comment:
Oh when I gave you the example I said should look like this but i had not tested it. So now i did test it and it does not work. 

> -G, --get When used, this option will make all data specified with -d, --data, --data-binary or --data-urlencode to be used in an HTTP GET request instead of the POST request that otherwise would be used. The data will be appended to the URL with a '?' separator.

So this version works but raises an error:

```bash
curl --get 'http://localhost:7700/indexes/movies/search' \
    --data-urlencode 'q=Avengers' \
    --data-urlencode 'filters=release_date > 795484800'
```
Last problem that I have is the following error: 
`{"message":"a filter doesn't have a value to compare it with"}%`